### PR TITLE
Add Logface to the NBC & PNC and Nutrition modules

### PR DESCRIPTION
**Pull request by FrankApiyo**
This PR accomplishes the following: 

- Adds the LogFace to the **NBC & PNC** and **Nutrition** modules
- Adds the **NBC & PNC** and **Nutrition** modules to the side menu
- Makes the home page configurable via props and adds a home page for both the NBC & PNC and   Nutrition modules

### CODE REVIEW

**Comment by FrankApiyo** [comment]
What's left is adding the correct Icons to the new sub-menues and ensuring that only one sub-menu is active at a time when the a user goes to the `logface` url

**Comment by FrankApiyo** [comment]
We may not able to add the correct Icons right now because they are not easily downloadable from the `marvelapp` handoffs. @OluwatobiMayowa  should be able to help with this

**Comment by oluwatobim** [comment]
@FrankApiyo Currently working on this.

**Comment by oluwatobim** [comment]
@FrankApiyo They should be downloadable here: https://marvelapp.com/8ijdg2g/screen/62619111/handoff

File: .circleci/config.yml [hunk 1]
```diff
@@ -29,6 +29,9 @@ jobs:
           REACT_APP_ONADATA_USER_URL: "https://api.ona.io/api/v1/user.json"
           REACT_APP_ONADATA_CLIENT_ID: "9qoIMXO7AVcYvGCn8R2sYVTpg50INPhD5HwMWOLz"
```
Comment (moshthepitt) [inline .circleci/config.yml]: Are these real secrets?  If so, can we please remove them?  It is a public repo....

File: src/components/DataCircleCard/index.tsx [hunk 2]
```diff
@@ -1,7 +1,13 @@
 import React from 'react';
```
Comment (moshthepitt) [inline src/components/DataCircleCard/index.tsx]: Please remove unused imports here.

File: src/components/page/SideMenu/SubMenu/index.tsx [hunk 3]
```diff
@@ -49,6 +49,7 @@ export interface SubMenuProps {
   childNavs: PageLink[];
   collapsedModuleLabel: string;
   setCollapsedModuleLabel?: SetCollapsedModuleLabel;
+  customIcon?: any;
```
Comment (moshthepitt) [inline src/components/page/SideMenu/SubMenu/index.tsx]: Lets not use any here.  If you investigate the icons you'll see that their type is something like `React.FunctionComponent<React.SVGProps<SVGSVGElement>>`

File: src/constants.tsx [hunk 4]
```diff
@@ -198,3 +209,8 @@ export type SUPERSET_SMS_DATA_SLICE = typeof SUPERSET_SMS_DATA_SLICE;
 export const SUPERSET_PREGNANCY_ANALYSIS_ENDPOINT =
   'https://discover.ona.io/superset/dashboard/53/?standalone=true';
```
Comment (moshthepitt) [inline src/constants.tsx]: I still think that this should be an env variable.  The minute the slice id or superset URL changes you'll have to come and make a change here - this is more of a configurable value as opposed to a constant.

File: src/constants.tsx [hunk 5]
```diff
@@ -198,3 +209,8 @@ export type SUPERSET_SMS_DATA_SLICE = typeof SUPERSET_SMS_DATA_SLICE;
...
 export type SUPERSET_PREGNANCY_ANALYSIS_ENDPOINT = typeof SUPERSET_PREGNANCY_ANALYSIS_ENDPOINT;
+
+// csv export links
+export const SUPERSET_PREGNANCY_DATA_EXPORT =
+  'https://discover.ona.io/superset/explore_json/?form_data={"slice_id":2263}&csv=true';
```
Comment (moshthepitt) [inline src/constants.tsx]: This should be an configurable env var

File: src/containers/LogFace/index.tsx [hunk 6]
```diff
@@ -111,21 +113,33 @@ export class LogFace extends React.Component<PropsInterface, State> {
...
+        });
+      },
+      pageLimit: 5,
+      pageNeighbours: 3,
+      previousLabel: 'previous',
+      startLabel: 'first',
+      totalRecords: this.props.smsData.length,
+    };
     const data = this.state.filteredData.sort((a: FlexObject, b: FlexObject) => {
       return (new Date(b.EventDate) as any) - (new Date(a.EventDate) as any);
```
Comment (moshthepitt) [inline src/containers/LogFace/index.tsx]: Why does a date have to be of type `any`?

Comment (FrankApiyo) [reply src/containers/LogFace/index.tsx]: Removed and gone

**Comment by moshthepitt** [comment]
@KipSigei based on our conversation, lets us create an issue to move the Superset configs to env vars later, and leave them as is now.

File: .circleci/config.yml (hunk 1 above)
Comment (FrankApiyo) [reply .circleci/config.yml]: I think we should consider changing the Oath Client IDs for this app.

File: README.md [hunk 7]
```diff
@@ -43,6 +43,9 @@ Run the tests
 yarn test
 ```
 
+**note: ** while running tests ensure you comment out `REACT_APP_OPENSRP_CLIENT_ID` and `REACT_APP_ONADATA_CLIENT_ID` from your .env file.
+also, ensure these are not pushed to github.
+
```
Comment (FrankApiyo) [inline README.md]: I thought adding this will help future me never make this mistake again.

**Comment by FrankApiyo** [comment]
This PR has been recreated here: https://github.com/OpenSRP/opensrp-web/pull/93

**Pull request closed without merging**